crates/iceberg-catalog/src/api/management/mod.rs

```diff
@@ -37,6 +35,109 @@ pub mod v1 {
                         },
                     ),
                 )
+                // List all projects
+                .route(
+                    "/project",
+                    get(
+                        |AxumState(api_context): AxumState<ApiContext<State<A, C, S>>>,
+                         Extension(metadata): Extension<RequestMetadata>| {
+                            Self::list_projects(api_context, metadata)
+                        },
+                    ),
+                )
+                
+                .route(
+                    "/project/:project_id/warehouses",
```

Comment:
move to top level with query param? e.g. `GET /warehouse?project_id={some_id}` that way we're consistent with the rest of the endpoints here

Reply:
Done